src/AppBundle/Repository/AutoEditsRepository.php

```diff
@@ -323,13 +347,12 @@ public function getToolCounts(
      * Get SQL for getting counts of known automated tools used by the given user.
      * @see self::getAutomatedCounts()
      * @param Project $project
-     * @param User $user
```

Comment:
If the user parameter is unused, it should be removed, but how is this related to the rest of the patch? Suggest doing separately

Reply:
It's the XTools way, hehe :) By that I mean I largely work alone here, so little null fixes like this get combined with bigger PRs to save me time. Indeed $user is unused, I'll modify the commit message accordingly, same with below.